domains/norelock.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "norelockk",
+        "email": "norbert.podbieglyyy@gmail.com"
+    },
+    "record": {
+        "A": ["3.120.151.77"]
```

Comment:
i cannot reach your website, keeps on loading

Reply:
I'll do after sleep the change.

Reply:
@norelockk site still isnt accessible